index.html

```diff
@@ -1,122 +1,94 @@
 <html>
-  <head>
-    <title>Fuck My Party - Role-Playing Tools for Jaded Players</title>
-    <script async type='text/javascript' src='fmp.js'></script>
-    <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic' />
-    <style type='text/css'>
-    body {
-      font: 1.2em Roboto;
-      margin: 0;
-      padding: 50px;
-    }
-    h1, h2, h3, h4, h5, h6 {
-      color: #EE7552;
-    }
-    a {
-      color: #397697;
-      text-decoration: none;
-    }
-    a:hover {
-      text-decoration: underline;
-    }
-    div.stat {
-      margin: 0;
-      padding: 0;
-      float: left;
-      padding-right: 10px;
-    }
-    div.stat span {
-      padding-left: 5px;
-    }
-    div.stat strong {
-      cursor: help;
-      border-bottom: 1px dashed #333;
-    }
-    div.clear {
-      clear: both;
-    }
-    .error {
-      color: red;
-    }
```

Comment:
Consider adding the .error class back in since it provides feedback when a field is left empty or invalid.